build/python/backend/Dockerfile

```diff
@@ -13,18 +13,24 @@ RUN apt-get -qq update && \
     apt-get install --no-install-recommends -qq \
     automake \
     build-essential \
+    ca-certificates \
     cmake \
     curl \
     gcc \
     git \
+    g++ \
     libglu1-mesa \
+    libleptonica-dev \
+    libpango1.0-dev \
     libtool \
     make \
     swig \
+    tesseract-ocr-eng \
     python3-dev \
     python3-pip \
     python3-wheel \
     pkg-config \
+    wget \
```

Comment:
Can we now uninstall some of these packages (see L138 and below). Removing dependencies reduces image size and the vulnerability footprint.

Then we should be g2g

Reply:
removed `wget` and `g++` 👍